src/uptane/initialize.cc

```diff
@@ -218,8 +236,6 @@ InitRetCode Repository::initEcuRegister() {
 
 // Postcondition: "ECUs registered" flag set in the storage
 bool Repository::initialize() {
-  if (storage.loadEcuRegistered()) return true;
-
   // if device has aready been registered, just create new device and reprovision
   //   it should be changed/fixed when the backend supports registration acknowledgement
```

Comment:
What do you mean in this comment by "create new device and reprovision"? Is that accurate? It looks to me like if everything is already initialized, this function will just reload everything, but it won't change the device ID or keys.